components/event-tile.jsx

```diff
@@ -29,6 +30,8 @@ export default class EventTile extends PureComponent {
         let sender = this.props.mxEvent.sender;
         let avatarUrl = sender.getAvatarUrl(this.props.homeserver, 32, 32, 'scale', false);
         let {name, userId} = sender;
+        let fmtBody = this.props.mxEvent.event.content.format == 'org.matrix.custom.html'
+            && this.props.mxEvent.event.content.formatted_body ? this.props.mxEvent.event.content.formatted_body : null;
```

Comment:
Looks like this whole line can be replaced with `this.props.mxEvent.event.content.formatted_body`?

Reply:
Yeah, I just wanted to check for the format to be `org.matrix.custom.html` because matrix-react-sdk code also does that sometimes. I'll clean this up.